src/Items/ItemArmor.h

```diff
@@ -114,6 +114,7 @@ class cItemArmorHandler:
 			case E_ITEM_LEATHER_CAP:
 			case E_ITEM_LEATHER_PANTS:
 			case E_ITEM_LEATHER_TUNIC:
+			case E_ITEM_ELYTRA:
```

Comment:
No phantom membrane? This at least needs a TODO.

Reply:
I thought adding it but we don't even have membrane in item lists. I added todo with some basic info.